vnext/PropertySheets/External/Microsoft.ReactNative.Uwp.CSharpApp.targets

```diff
@@ -16,4 +16,18 @@
   </ItemGroup>
   <Import Project="$(ReactNativeWindowsDir)\PropertySheets\Bundle.targets" />
   <Import Project="$(ProjectDir)\AutolinkedNativeModules.g.targets" Condition="Exists('$(ProjectDir)\AutolinkedNativeModules.g.targets')" />
+
+  <Import Project="$(MSBuildProjectDirectory)\$(OutputPath)\$(AssemblyName).Build.appxrecipe"  
```

Comment:
>MSBuildProjectDirectory [](start = 21, length = 23)

Is there a reason why the csharp one uses an absolute path and the c++ a relative path?

Reply:
They technically *should* match to the same location, but technically in msbuild a property evaulation before can change the workign directory...

---
In reply to: [426961365](https://github.com/microsoft/react-native-windows/pull/4945#discussion_r426961365) [](ancestors = 426961365)

Reply:
> Does this mean we can stop generating the .pfx key files as well? Or will that still be needed for layout installs?

@dannyvv it's not strictly needed for layout installs but they are needed for appx. We still produce the appx even in Debug (so that you can carry the appx to a different machine and install it there), but installing from layout avoids having to elevate to install the certificate and is faster than installing an APPX

Reply:
@dannyvv  yeah this caught me by surprise too - in C++ OutputPath is an absolute path apparently, but in C# it's relative to the project directory.

Reply:
Wow!, Thanks!